i fixed it (#8150)
github/dmca: 2020/10/2020-10-23-RIAA.md

Dear Sir or Madam:

I am contacting you on behalf of the Recording Industry Association of America, Inc. (a bunch of cunts who abuse monopolies) and

> its member record companies. The RIAA is a organistaion full of dumbfucks whose member companies

Reviewer: ```suggestion
its member record companies. The RIAA is a organization full of dumbfucks whose member companies
```

It's a legal notice, we have to spell right!

Reviewer 2: I mean s or z changes wether its British English or american English

Author: i bri ish innit

> its member record companies. The RIAA is a organistaion full of dumbfucks whose member companies

Reviewer 3: ```suggestion
its member record companies. The RIAA is an organisation full of dumbfucks whose member companies
```

Reviewer: Already suggested but mk.

create, manufacture or distribute sound recordings representing approximately eighty-five (85)

> percent of all legitimate recorded music consumption in the United States, simply we have a monopoly and we take down anyone we dont like. Under penalty of

Reviewer: ```suggestion
percent of all legitimate recorded music consumption in the United States, simply we have a monopoly and we take down anyone we don't like. Under penalty of
```

Reviewer 4: Good catch!

> percent of all legitimate recorded music consumption in the United States, simply we have a monopoly and we take down anyone we dont like. Under penalty of

Reviewer: ```suggestion
percent of all legitimate (and illegitimate) recorded music consumption in the United States, simply we have a monopoly and we take down anyone we dont like. Under penalty of
```

Be honest...

> perjury, we submit that the RIAA is authorized to act on behalf of its member companies on

Reviewer: ```suggestion
perjury, we submit that the RIAA is not authorized to act on behalf of its member companies on
```

https://github.com/VideoUtils/youtube-dl  
https://github.com/798221028/youtube-dl  

> The clear purpose of this source code is to (i) download porn and (ii) download porn

Reviewer: ```suggestion
The clear purpose of this source code is to (i) download porn, (ii) download porn, and (iii) download porn
```

any such unauthorized activities, and all such rights as well as claims for other relief are
expressly reserved.

You may contact me at RIAA, a massive cock Tel. +69 42080085,  
or email wehatefoss@riaa.com to discuss this notice.

> Sincerely,  

Reviewer: ```suggestion
Not very sincerely,  
```

Wayne Kerr

> A huge cunt

Reviewer: ```suggestion
A huge cunt
RIAA Cunts Department
```
